gobblin-modules/gobblin-http/src/main/java/gobblin/writer/AvroHttpWriterBuilder.java

```diff
@@ -36,4 +42,16 @@ public AvroHttpWriterBuilder fromConfig(Config config) {
     this.responseHandler = new HttpResponseHandler();
     return this;
   }
+
+  public String getLimiterKey () throws NotConfiguredException {
+    try {
+      Config config = ConfigBuilder.create().loadProps(state.getProperties(), CONF_PREFIX).build();
+      String urlTemplate = config.getString(HttpConstants.URL_TEMPLATE);
+      URL url = new URL(urlTemplate);
+      log.info("Get limiter key [" + url.getProtocol() + url.getPath() + "]");
```

Comment:
Let's discuss what the correct key is. I don't think the path is useful for policying.